lib/driver.js

```diff
@@ -314,6 +321,8 @@ class XCUITestDriver extends BaseDriver {
       await retryInterval(15, 1000, async () => {
         log.debug('Sending createSession command to WDA');
         try {
+          // This is needed to put the '/status' output into the cache
+          await this.getStatus();
```

Comment:
If you want to save a call to WDA, it would be relatively simple to pass the status back from launch, since in `waitForStart` we poll `/status`.